### PR TITLE
update help window, remove empty fields

### LINKED ISSUE #138: [PE-D][Tester C] Screenshot for Help is not updated and Copy URL button is not flushed to the bottom right corner
## In Application

Notice that screenshot provided in UG is not updated to reflect the changes in the help screen shown in application. Particularly, 1 thing that stood out is that the Copy URL button is not flushed to the bottom right corner

Additionally, not all features available are listed in the help screen.

-------------
Labels: `type.DocumentationBug` `severity.VeryLow`
original: Tan-Jia-Rong/ped#2

### LINKED ISSUE #150: [PE-D][Tester C] Remove command works without specifying any field
## In Application

UG denotes that remove requires 1 field. However, exception is not thrown when only `remove` is passed in as argument.

Hence, remove does not work as intended and is categorised under functionality bug.

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: Tan-Jia-Rong/ped#6

### PR DESCRIPTION
**Pull request by farisanadia**
Updated so remove command now gives error for empty fields
Help window copy button pushed down

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/RemoveCommand.java [hunk 1]
```diff
@@ -294,5 +321,26 @@ public void setTags(Set<Tag> tags) {
...
+                return false;
+            }
+
+            // state check
+            RemovePersonDescriptor r = (RemovePersonDescriptor) other;
+
+            return getName().equals(r.getName())
+                    && getPhone().equals(r.getPhone())
+                    && getLessonPlan().equals(r.getLessonPlan())
+                    && getTags().equals(r.getTags());
```
Comment (cfyjoanna) [inline src/main/java/seedu/address/logic/commands/RemoveCommand.java]: can remove all the unnecessary fields

Comment (farisanadia) [reply src/main/java/seedu/address/logic/commands/RemoveCommand.java]: yup thats the next one hwne i add tests

Comment (cfyjoanna) [reply src/main/java/seedu/address/logic/commands/RemoveCommand.java]: i meant to remove like phone, name, lessonplan entirely from removepersondescriptor not remove the whole equals method 😭😭 

just compare the relevant ones like homework grade attendance in equals method only

**Review by cfyjoanna — APPROVED** [review]
LGTM

**Pull request merged**